modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -51,52 +46,65 @@
 public class OperationMgtService {
 
     private static Log log = LogFactory.getLog(OperationMgtService.class);
+    private static final String ACCEPT = "Accept";
 
     @PUT
     @Produces(MediaType.APPLICATION_JSON)
```

Comment:
While we allow any type of response via the accept header,  why do we also limit response to JSON?